scripts/process_data.py

```diff
@@ -129,7 +131,11 @@ def main(self) -> None:
         # Copy and downscale images
         if not self.skip_image_processing:
             # Copy images to output directory
-            num_frames = process_data_utils.copy_images(self.data, image_dir=image_dir, verbose=self.verbose)
+            image_rename_map_paths = process_data_utils.copy_images(
+                self.data, image_dir=image_dir, verbose=self.verbose
+            )
+            image_rename_map = OrderedDict((a.name, b.name) for a, b in image_rename_map_paths.items())
```

Comment:
Isn't `image_rename_map_paths` already an `OrderedDict`?

Reply:
It is. Here the rename_map takes the filenames as strings.

Reply:
just wondering: why not use a regular dict?

Reply:
No reason, switching to dict.